tests/testthat/test-get_people.R

```diff
@@ -1,13 +1,42 @@
 skip_on_ci()
+load("infotest.rda")
+instr_team <- Microsoft365R::get_team("Instructors")
 
 test_that("People are identified as Instructors team members", {
-  load("infotest.rda")
-  instr_team <- Microsoft365R::get_team("Instructors")
   team_members <- get_people(infotest,
                              alert = c("Dani Bodor", "Mateusz Kuzak"),
                              instr_team)
 
   expect_true(is.list(team_members))
-  expect_equal(length(team_members), 4)
+  expect_equal(length(team_members), 4) ## 2 teachers in infotest + 2 people in alert
   expect_true("ms_team_member" %in% class(team_members[[1]]))
 })
+
+test_that("When people cannot be identified, the program reports this.", {
+  errorps <- "Person Notemployed"
+
+  expect_warning(get_people(infotest,
+                            alert = errorps,
+                            instr_team),
+                 regexp = "Cannot find team member Person Notemployed")
+
+})
+
+test_that("Some people who were not alerted in the past are identified correctly.",{
+
+  team_members <- get_people(infotest,
+                             alert = c("Heli Jarvenpaa",
```

Comment:
wondering if Heli will soon disappear from this since she's left the Center?

Reply:
People stay remarkably long, I've found. We can delete her name when it causes the test to fail in the future — or we can chuck it already. I used it mostly to confirm that the function worked and could deal with weird characters.